ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -60,7 +60,8 @@ class RuntimeWorker(_worker.PeriodicWorkerThread):
 
     FLUSH_INTERVAL = 10
 
-    def __init__(self, statsd_client, flush_interval=FLUSH_INTERVAL):
+    def __init__(self, statsd_client, flush_interval=None):
+        flush_interval = self.FLUSH_INTERVAL if flush_interval is None else flush_interval
```

Comment:
why?

Reply:
We did this because we had trouble finding a way to override the flush interval in a test case.

This isn't ideal and should likely be refactored into something else.


Maybe:

```
class Tracer:
    def configure(self, ... _runtime_flush_interval=10):
        pass
```


This is what we are doing that required us to make this change:
https://github.com/DataDog/dd-trace-py/blob/144d10d69ff06456bce0320a3f0418095dc773f5/tests/internal/runtime/test_runtime_metrics.py#L53-L62

Reply:
Understood. But in this case, the test should just create its own custom worker, not overriding like this. :(

Reply:
Fair enough.

Yeah `configure` should allow to customize that ultimately.

Reply:
yeah, I think the main think we wanted to test was end-to-end letting the tracer configure the runtime worker, did the right thing